test/built-ins/Simd/Simd.round_trip.js

```diff
@@ -0,0 +1,31 @@
+//  Copyright (C) 2015
+//
+//  This software is provided 'as-is', without any express or implied
+//  warranty.  In no event will the authors be held liable for any damages
+//  arising from the use of this software.
+//
+//  Permission is granted to anyone to use this software for any purpose,
+//  including commercial applications, and to alter it and redistribute it
+//  freely, subject to the following restrictions:
+//
+//  1. The origin of this software must not be misrepresented; you must not
+//     claim that you wrote the original software. If you use this software
+//     in a product, an acknowledgment in the product documentation would be
+//     appreciated but is not required.
+//  2. Altered source versions must be plainly marked as such, and must not be
+//     misrepresented as being the original software.
+//  3. This notice may not be removed or altered from any source distribution.
+
+/*---
+description: Test round trip.
+includes: [simdUtilities.js]
+---*/
+
+testSimdFunction('Float32x4 Int32x4 round trip', function() {
+  // NaNs should stay unmodified across bit conversions
+  var m = SIMD.Int32x4(0xFFFFFFFF, 0xFFFF0000, 0x80000000, 0x0);
+  var m2 = SIMD.Int32x4.fromFloat32x4Bits(SIMD.Float32x4.fromInt32x4Bits(m));
+  // NaNs may be canonicalized, so these tests may fail in some implementations.
+  equalInt32x4(m, m2);
+});
+
```

Comment:
The point of the test is to check that the bits match, so not doing bit matching would make the test pointless